Nomic_Analysis/README.md

```diff
@@ -0,0 +1,8 @@
+# Nomic nELISA panel analysis
```

Comment:
Based on this README, maybe you can remove that README from the `Data` folder so you can have all of the cleaning information here.